projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/stock-notification.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  subscribeStockNotification,
+  enableNotificationChannel,
+  clickNotifyMeBtn,
+  navigateToPDP,
+  unsubscribeStockNotification,
+} from '../../notification';
+import { TabElement } from '../tabbing-order.model';
+import { verifyTabbingOrder } from '../tabbing-order';
+
+const containerSelector = '.ProductDetailsPageTemplate cx-stock-notification';
+
+export function stockNotificationNotLoginTabbingOrder(config: TabElement[]) {
+  cy.visit('/product/872912');
```

Comment:
In a11y tests, we use one test product (for simplicity): `export const testProductUrl = '/product/779841';`. Either use this test product in the whole file, or take the product ID from all of the functions and place it in some local variable in this file.

Reply:
Fixed.